### ISSUE TITLE
Create dedicated website for the project

### ISSUE THREAD

**Issue by konrad** [OKScienceDE/Open_Science_101#16]

**Comment by merriam** [comment]
Hi Konrad.   I assume you want a website separate from GitHub repository?   Could you elaborate on what should be included and what specific audience needs should be served?

**Comment by konrad** [comment]
Correct, a separate website with its own domain was mentioned during the initial brain storming. But this will be something for a later phase. We first need find a consensus of the audience and scope of the collection. See issue #6 which is crucial for the whole operation. If you have any opinion regarding that please free to put that there.

**Comment by aleimba** [comment]
I'm also fine with GitHub pages, as long as the content can be pulled from this repo directly and synced.

**Comment by konrad** [comment]
GitHub page would also be fine for me (e.g. using [Pelican](http://docs.getpelican.com/en/3.3.0/tips.html) or [Jekyll](https://help.github.com/articles/using-jekyll-as-a-static-site-generator-with-github-pages/))
